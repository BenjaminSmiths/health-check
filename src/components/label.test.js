import React from "react";
import ReactDOM from "react-dom";
import Label from "./label";
import renderer from 'react-test-renderer';

it('renders a label', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Label value='this is required' />, div);
});

it('lable can have children', () => {
    const tree = renderer.create(
        <Label value='this is required' ><div id="child">hello world</div></Label>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});