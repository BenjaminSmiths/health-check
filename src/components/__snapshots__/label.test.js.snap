// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lable can have children 1`] = `
<div
  style={
    Object {
      "alignContent": "space-between",
      "alignItems": "center",
      "color": "white",
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <h3>
    this is required
  </h3>
  <div
    id="child"
  >
    hello world
  </div>
</div>
`;
